### PR TITLE
Psl 228 convert rpc help str to r str

### PR DESCRIPTION
**Pull request by tanlemin91**

### CODE REVIEW

File: src/mnode/rpc/masternode.cpp [hunk 1]
```diff
@@ -540,22 +554,27 @@ UniValue masternode(const UniValue& params, bool fHelp)
...
-                               "    "extP2P:port"       (string) (required) The address and port of the Master Node's Kademlia point\n"
-                               "    "passphrase"        (string) (required) passphrase for new PastelID\n"
-                               "    "txid"              (string) (optional) id of transaction with the collateral amount\n"
-                               "    "index"             (numeric) (optional) index in the transaction with the collateral amount\n"
-                               "\nCreate masternode configuration\n")" +
-                                   HelpExampleCli("masternode make-conf",
-                                                  R"("myMN" "127.0.0.1:9933" "127.0.0.1:4444" "127.0.0.1:5545" "bc1c5243284272dbb22c301a549d112e8bc9bc454b5ff50b1e5f7959d6b56726" 4)") +
-                                   "\nAs json rpc\n" + HelpExampleRpc("masternode make-conf", R"(""myMN" "127.0.0.1:9933" "127.0.0.1:4444" "127.0.0.1:5545" "bc1c5243284272dbb22c301a549d112e8bc9bc454b5ff50b1e5f7959d6b56726" 4")")
+                R"("masternode make-conf "alias" "mnAddress:port" "extAddress:port" "extP2P:port" "passphrase" "txid" "index"
```
Comment (akobrin1) [inline src/mnode/rpc/masternode.cpp:557]: Shift all lines left to remove whitespace at the beggining

File: src/wallet/rpcwallet.cpp [hunk 2]
```diff
@@ -1351,30 +1394,33 @@ UniValue listreceivedbyaddress(const UniValue& params, bool fHelp)
...
-            + HelpExampleCli("listreceivedbyaddress", "")
-            + HelpExampleCli("listreceivedbyaddress", "6 true")
-            + HelpExampleRpc("listreceivedbyaddress", "6, true, true")
-        );
+R"(listreceivedbyaddress ( minconf includeempty includeWatchonly)
+
+List balances by receiving address.
+
+Arguments:
+1. minconf       (numeric, optional, default=1) The minimum number of confirmations before payments are included.
```
Comment (akobrin1) [inline src/wallet/rpcwallet.cpp]: can align argument descriptions

File: src/wallet/rpcwallet.cpp [hunk 3]
```diff
@@ -2103,8 +2180,8 @@ UniValue walletpassphrasechange(const UniValue& params, bool fHelp)
 
     if (strOldWalletPass.length() < 1 || strNewWalletPass.length() < 1)
         throw runtime_error(
-            "walletpassphrasechange <oldpassphrase> <newpassphrase>\n"
-            "Changes the wallet passphrase from <oldpassphrase> to <newpassphrase>.");
+            R"(walletpassphrasechange <oldpassphrase> <newpassphrase>
+            Changes the wallet passphrase from <oldpassphrase> to <newpassphrase>.)");
```
Comment (akobrin1) [inline src/wallet/rpcwallet.cpp]: trim whitespace from the left, otherwise second line will be shifted

File: src/wallet/rpcwallet.cpp [hunk 4]
```diff
@@ -2160,35 +2239,37 @@ UniValue encryptwallet(const UniValue& params, bool fHelp)
     string enableArg = "developerencryptwallet";
     auto fEnableWalletEncryption = fExperimentalMode && GetBoolArg("-" + enableArg, false);
 
-    std::string strWalletEncryptionDisabledMsg = "";
+    string strWalletEncryptionDisabledMsg = "";
```
Comment (akobrin1) [inline src/wallet/rpcwallet.cpp]: please remove this string initialization to "" (empty by default)

File: src/wallet/rpcwallet.cpp [hunk 5]
```diff
@@ -2200,16 +2281,16 @@ UniValue encryptwallet(const UniValue& params, bool fHelp)
...
     SecureString strWalletPass;
     strWalletPass.reserve(100);
     strWalletPass = params[0].get_str().c_str();
 
     if (strWalletPass.length() < 1)
         throw runtime_error(
-            "encryptwallet <passphrase>\n"
-            "Encrypts the wallet with <passphrase>.");
+            R"(encryptwallet <passphrase>
+            Encrypts the wallet with <passphrase>.)");
```
Comment (akobrin1) [inline src/wallet/rpcwallet.cpp]: trim left (2nd line)

**Review by akobrin1 — CHANGES_REQUESTED** [review]
just few small corrections needed

File: src/mnode/rpc/masternode.cpp (hunk 1 above)
Comment (tanlemin91) [reply src/mnode/rpc/masternode.cpp:557]: done thanks

File: src/wallet/rpcwallet.cpp (hunk 3 above)
Comment (tanlemin91) [reply src/wallet/rpcwallet.cpp]: done thanks

File: src/wallet/rpcwallet.cpp (hunk 5 above)
Comment (tanlemin91) [reply src/wallet/rpcwallet.cpp]: done thanks

**Review by akobrin1 — APPROVED** [review]
noticed some quotes missing, but I think we can add them in the next check-in.... just don't need to run CircleCi one more time because of this minor problem.
let's not forget about it

**Pull request merged**